apps/docs/tests/e2e/iconsVisible.cy.ts

```diff
@@ -0,0 +1,17 @@
+import { IconVisiblePageObject } from '../support/pageObject/IconVisible.PageObject';
+
+const iconVisible = () => new IconVisiblePageObject();
+
+describe('Icons visible', () => {
+  it('vue icons check', () => {
+    cy.visit('/vue/components/iconbase.html');
+    iconVisible().checkIconVisibilityInIframe();
+  });
+
+it.skip('react icons check', () => { //waiting for react fix
```

Comment:
I believe icons works now https://docs.storefrontui.io/v2/react/components/iconbase.html so `skip` can be removed